docs/UserGuide.md

```diff
@@ -97,46 +97,50 @@ Expressions without `...` at the end takes the last parameter as the argument wh
 ### Adding a tag with filepath : `tag`
 
 Tags a file with a unique tag name.<br>
-You can add a nickname to a file. Optionally, you can add some labels to the tag for categorisation.
-The `FILE_PATH` field can accept both absolute and relative file path from your current directory in HelloFile.
+You can add a nickname to a file. This make it easier to manage only some specific files that you want such as frequently used files, etc.
+
+Please take note of the following:
+* You can add any number of labels to the tag (including 0).
+* The `FILE_PATH` field can accept both absolute and relative file path from your current directory specified in the internal File Explorer of HelloFile.
+* You can tag a file without access permission.
 
 Format: `tag t>TAG_NAME f>FILE_PATH [l>LABEL]...`
 
 ![tag](images/screenshots/tag_command.png)
 
 Examples:
-* `tag t>Users f>C:\Users` (Adds a tag with tag name `Users`,absolute file path `C:\Users`, and no label)
-* `tag t>Users f>C:\Users l>Important` (Adds a tag with tag name `Users`, absolute file path `C:\Users`, and label name `Important`)
-* `tag t>Users f>.\Users` (Adds a tag with tag name `Users`, relative file path `Users`, and no label)
-* `tag t>Users f>.\Users l>folder l>readonly` (Adds a tag with tag name `Users`, relative file path `Users`, and multiple lables
-with label name `folder` and `readonly`)
+* `tag t>Users f>C:\Users` (Adds a tag with tag's nickname `Users` that points to `C:\Users` with no label to the tag list)
```

Comment:
personally I feel maybe can change "tag's nickname" to "the nickname", since tag is repeated several times here. But it's just personal opinion

Reply:
Noted. Thanks!